public/style.css

```diff
@@ -93,6 +98,15 @@ input:focus, select:focus {
   border-color: #333333;
 }
 /*Uploading-photo animation*/
+/* divi? */
+.divi {
```

Comment:
Why is this in the pull? I thought it was for the spinner? Also I think it has been deleted